code/modules/mob/dead/new_player/new_player.dm

```
@@ -41,8 +41,22 @@
 	if(!client)
 		return
 
+	if(href_list["manifest"])
+		ViewManifest()
+		return
+
 	if(client.restricted_mode)
-		return FALSE
+		if(href_list["verify"])
+			show_otp_menu()
+			return TRUE
+
+		if(href_list["link_to_discord"])
+			var/_link = CONFIG_GET(string/panic_bunker_discord_link)
+			if(_link)
+				src << link("https://discord.daedalus13.net/")
```

Throw an error message instead of falling back to us.